### ISSUE TITLE
How to access the activations of a certain layer?

### ISSUE THREAD

**Issue by thecyclone** [ultralytics/yolov5#4815]

**Comment by glenn-jocher** [comment]
@thecyclone I don't understand what you are asking. An activation is just a function, i.e. nn.SiLU:
https://pytorch.org/docs/stable/generated/torch.nn.SiLU.html

They don't have any parameters and there's nothing to access, it's just an equation.

**Comment by thecyclone** [comment]
@glenn-jocher sorry for the confusion. I meant how can I get the output of a certain layer before the final predictions?

**Comment by glenn-jocher** [comment]
@thecyclone all pytorch models have forward methods that contain outputs from every layer:
https://github.com/ultralytics/yolov5/blob/621b6d5ba80707ca98242dd7c71d738e5594b41e/models/yolo.py#L138-L157

**Comment by thecyclone** [comment]
@glenn-jocher, I am still confusd.
`model = torch.hub.load('ultralytics/yolov5', 'yolov5m', pretrained=True)`
`imgs = ['https://ultralytics.com/images/zidane.jpg']  # batch of images`
`results = model(imgs)`

I tried using model.forward_once, but I got the following error.
`AttributeError: 'AutoShape' object has no attribute 'forward_once'`

More specifically, my question is how can I get outputs from any layer within the model when I do a forward pass? Thank you again for your help.
